client/my-sites/domains/domain-management/domain-connect-mapping/index.jsx

```diff
@@ -115,7 +115,7 @@ class DomainConnectMapping extends React.Component {
 					<div>
 						<p>
 							{ translate(
-								'Your domain provider supports automattically configuring your ' +
+								'Your domain provider supports automatically configuring your ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Your domain provider supports automattically configuring your domain to use it with WordPress.com.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).